src/main/webapp/jsp/instructorFeedbackResultsTop.jsp

```diff
@@ -14,8 +14,13 @@
       showAll="${showAll}" />
 </c:if>
 <br>
+<div style="margin-top:10px">
 <t:statusMessage statusMessagesToUser="${data.statusMessagesToUser}" />
-<br>
+</div>
+<c:if test="${not (noResponses and data.allSectionsSelected and showAll)}">
+    <r:expandCollapseButton filterPanel="${data.filterPanel}"
+                    showAll="${showAll}" />
```

Comment:
Fix the indentation: line 21 should be 2 spaces from the start and line 22 should be 6 spaces from the start.